account_journal_report/tests/test_account_journal_report.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestAccountJournalReport(TransactionCase):
+    """
+        Tests for Account Journal Report.
+    """
+    def setUp(self):
+        super(TestAccountJournalReport, self).setUp()
+
+        self.res_users_model = self.env['res.users']
+        self.company = self.env.ref('base.main_company')
+        self.g_account_user = self.env.ref('account.group_account_user')
+
+        self.user = self._create_user('user_1', [self.g_account_user],
+                                      self.company).id
+
+        self.report_model = \
+            self.env['report.account_journal_report.journal_ledger']
+        self.wiz = self.env['account.journal.entries.report']
+        self.report_name = 'account_journal_report.journal_ledger'
+        self.report_xlsx_name = 'account_journal_report.journal_ledger_xlsx'
+        self.report_title = 'Journal Ledger'
+
+    def _create_user(self, login, groups, company):
```

Comment:
You don't need to create a new user for this, as tests are run as admin, and I see no use of this user for testing any permission.